mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -183,6 +183,20 @@ def fields_view_get(
             result['fields'] = all_fields
         return result
 
+    def read(self, cr, uid, ids, fields, context=None):
+        """ Without this call, dynamic fields defined in fields_view_get()
+            generate a log warning, i.e.:
```

Comment:
I don't understand how to add dynamic fields in fields_view_get, do you have some code sample ? 
thanks.